maps/site53/items/clothing/solgov-accessory.dm

```diff
@@ -57,7 +57,7 @@ specialty pins
 
 /obj/item/clothing/accessory/solgov/speciality/pilot
```

Comment:
See if you can delete this entirely, we don't have spaceship piloting.

Reply:
fixed